frontend/src/component/CustomToolboxContainer.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import { connect } from 'react-redux'
+import appFactory from '../util/appFactory'
+import PropTypes from 'prop-types'
+import { usePublishLifecycle } from 'tracim_frontend_lib'
+
+const CustomToolboxContainer = (props) => {
+  usePublishLifecycle('TOOLBOX', {
```

Comment:
I wonder if these strings should be constant. Maybe an "component" object in frontend_lib/src/customHooks.jsx.

Reply:
Indeed, this could be useful, but since they are designed to be used once in the code (no duplicate components) is it really necessary or it will add a handful of const strings which the sole goal is to be used once ?

I don't really know what is better, but I'd personally prefer to keep it like it is.

Reply:
I had thought more about making sure it was consistent (string with capital letters), but I don't have a strong opinion, so no problem leaving it that way if you prefer.

Reply:
Ideally every string are constants indeed. That case is specific since the string isn't supposed to be used somewhere else in tracim's code and will be used by another app (custom toolbox). So I don't mind the string to be kept that ways